src/NinjaTools.FluentMockServer.API/Proxy/Visitors/IVisitor.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading;
+
+namespace NinjaTools.FluentMockServer.API.Proxy.Visitors
+{
+    public interface IVisitor
+    { }
+
+    public interface IVisitor<TVisitable>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'in' keyword to parameter 'TVisitable' to make it 'contravariant'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)